hat/assets/js/apps/Iaso/domains/orgUnits/components/TreeView/OrgUnitTreeviewModal.tsx

```diff
@@ -147,7 +175,7 @@ const OrgUnitTreeviewModal = ({
         if (multiselect) {
             onConfirm([]);
         } else {
-            onConfirm(null);
+            onConfirm(undefined);
```

Comment:
I could learn something here, what is the rationale behind using `undefined` rather than `null` here?